src/typeOfAnimals/Animal.java

```diff
@@ -0,0 +1,8 @@
+package typeOfAnimals;
+
+public abstract class Animal {
+    protected String isMoving;
```

Comment:
the type of isMoving should be boolean. The is moving express an question that can be either true or false